test-integration/Microsoft.Azure.WebJobs.Extensions.OpenApi.TestApp/Models/DictionaryObjectModel.cs

```diff
@@ -0,0 +1,15 @@
+using System.Collections.Generic;
+
+namespace Microsoft.Azure.WebJobs.Extensions.OpenApi.TestApp.Models
+{
+    public class DictionaryObjectModel<TKey, TValue>
+    {
+        public Dictionary<TKey, TValue> Dictionary { get; set; }
```

Comment:
The name, `Dictionary` is not correct. If the name of `TValue` is `BlackWidows`, the property name of `Dictionary<TKey, TValue>` should also be `BlackWidows`, not `Dictionary`.